horovod/run/mpi_run.py

```diff
@@ -18,15 +18,21 @@
 import traceback
 import sys
 import os
+from enum import Enum
```

Comment:
Looks like unit tests are failing due to this import (requires adding `enum34` to `setup.py`), but since we don't seem to be using this anymore, should be okay to just remove it.